lib/capybara.rb

```diff
@@ -48,7 +48,7 @@ class << self
     # [ignore_hidden_elements = Boolean]  Whether to ignore hidden elements on the page (Default: true)  
     # [automatic_reload = Boolean]        Whether to automatically reload elements as Capybara is waiting (Default: true)  
     # [save_and_open_page_path = String]  Where to put pages saved through save_and_open_page (Default: Dir.pwd)  
-    #
+    # [first_default_waiting = Boolean]   Whether Node#first defaults to Capybara waiting behavior for at least 1 element to match (Default: false) 
```

Comment:
yeah -- the name I used isn't great -- but I don't think enable_waiting_by_first is great either since it seems to imply #first can't wait without that being true, which it can as long as you pass minimum, count, etc. - I'm open to other suggestions.